neopia/neosoco.py

```diff
@@ -439,6 +439,50 @@ def convert_scale(self, port='in1', omin=0, omax=255, tmin=0, tmax=100):
         else:
             raise TypeError
 
+    def check_color(self, port='in1', color='White'):
+        c_value = 0
+        if isinstance(port, str):
+            if port.lower() =='in1':
+                c_value = self.read(Neosoco.INPUT_1)
+            elif port.lower() =='in2':
+                c_value = self.read(Neosoco.INPUT_2)
+            elif port.lower() =='in3':
+                c_value = self.read(Neosoco.INPUT_3)
+            else:
+                raise ValueError('Wrong value of port')
+        else:
+            raise ValueError('Wrong value of port')
+        if isinstance(port, str):
+            if (c_value >= 10 and c_value <= 50):
+                if (color == 'White'): 

```

Comment:
Make the color value be lower by color.lower() to accept both uppercase and lowercase from user.